README.md

```diff
@@ -0,0 +1,265 @@
+# Lights Off!
+
+Lights Off! is a rendition of the original handheld game "Lights Out!", manufactured by Tiger Electronics. If you haven't played it before, you may well find Lights Off! to be a very addictive puzzle game.
+
+![LightsOff Screenshot](LightsOff.png)
+
+### How to Play
+
+The objective is to turn all the lights off by pushing the fewest number of buttons. Easier said than done. ;^)
+
+All 1050 puzzles from the original ("Classic") game have been included, but arranged a bit differently. The puzzles have been grouped into puzzle packs under the 'Puzzle Packs' menu. The original game's main puzzles are in the "Classic" pack. The rest are included from the manual for the original game and have been grouped by difficulty, with "Six Move Puzzles" being the easiest and "Fifteen Move Puzzles" being the hardest. Beginners are advised to attempt the Classic pack before trying the others. Note that you may switch from one puzzle pack to another and not lose the progress you have made. You may also re-play any puzzle you have already solved by choosing it from the 'Levels' menu. The Classic pack features 50 puzzles, and the others host 100 each.
```

Comment:
Maybe add a linebreak after "...being the easiest and "Fifteen Move Puzzles" being the hardest."
Use underscores to make the words in inverted commas and quotes italic, e.g. _Levels_ and _Classic_.
Not just here, but in the whole readme.

Reply:
Ok. Thanks.

Reply:
> Use underscores to make the words in inverted commas and quotes italic, e.g. Levels and Classic.

I don't understand. :( Can you explain?

Reply:
I'm an idiot, because my text was interpreted as markdow, and so you didn't see exactly what I wrote. One more try:

I mean, to have these words that are now in quotes stand out a bit more, use markdown formatting to make them italic instead of using quotes. Like `_Levels_` instead of  "Levels".
Oh, and apparently markdown does have tables (I actually used myself before...), see https://guides.github.com/features/mastering-markdown/

Reply:
Nvm. Got it.

Reply:
My screen didn't update until after I sent the above reply, but you had already explained again. Thanks!